mlflow/gateway/client.py

```diff
@@ -95,26 +96,25 @@ def get_route(self, name: str):
 
         return Route(**response)
 
-    def search_routes(self, search_filter: Optional[str] = None):
+    def search_routes(self, page_token: Optional[str] = None):
         """
         Search for routes in the Gateway. Currently, this simply returns all configured routes.
 
-        .. note::
-            Search is currently not implemented. Providing a `search_filter` term will raise an
-            exception. Leave the arguments empty to get a listing of all configured routes.
 
         :param search_filter: An optional filter to apply to the search. Currently not used.
         :return: Returns a list of all configured and initialized `Route` data for the MLflow
             Gateway Server. The return will be a list of dictionaries that detail the name, type,
             and model details of each active route endpoint.
         """
```

Comment:
silly nit: pydoc entry for page_token + remove entry for search_filter

Reply:
Thanks for the catch - updated docstrings :)